src/sr/discord_bot/rss.py

```diff
@@ -45,7 +45,7 @@ def create_embed(post: FeedParserDict) -> discord.Embed:
         description=text,
     )
 
-    if len(post.media_thumbnail) > 0:
```

Comment:
Question: Is it possible to have a `media_thumbnail` key, but it be empty?

Reply:
It could be, although Jekyll doesn't seem to handle it that way. Check the entries in https://studentrobotics.org/feed.xml for reference.